exercises/isbn-verifier/example.go

```diff
@@ -0,0 +1,61 @@
+package isbn
+
+import (
+	"errors"
+	"math"
+	"strconv"
+	"unicode"
+)
+
+func IsValidISBN(isbn string) bool {
+
+	// step 1
```

Comment:
These comments don't tell me much. It is self-evident that these are steps. Better would be why this step at this point rather than a label...